.travis.yml

```diff
@@ -1,19 +1,19 @@
 dist: bionic
 language: go
 go:
-  - 1.11.x
-  - 1.12.x
+  - 1.13.x
+  - 1.14.x
   - tip
 
 matrix:
   include:
-    - go: 1.12.x
+    - go: 1.13.x
       env:
         - RUNC_USE_SYSTEMD=1
       script:
         - make BUILDTAGS="${BUILDTAGS}" all
         - sudo PATH="$PATH" make localintegration RUNC_USE_SYSTEMD=1
-    - go: 1.12.x
+    - go: 1.13.x
```

Comment:
One thing I don't understand here (maybe just because it's slow Saturday) is why there is only a single versoin of go (such as `go: 1.13.x`) in this matrix.